pkg/transform/util/util.go

```diff
@@ -0,0 +1,24 @@
+package util
+
+import (
+	"fmt"
+	"strings"
+)
+
+type SecretDef struct {
+	SecretName string
+	DataKey    string
```

Comment:
I am not sure, this whole thing feels very disconnected to me. Why can't we keep this in the opencompose object? Like we read it as string but we don't need to save it as string, we can split it in the encoding itself.